src/map/instance_loader.cpp

```diff
@@ -44,6 +45,8 @@ CInstanceLoader::CInstanceLoader(uint8 instanceid, CZone* PZone, CCharEntity* PR
     zone = PZone;
     CInstance* instance = ((CZoneInstance*)PZone)->CreateInstance(instanceid);
 
+    petutils::LoadPetList();
```

Comment:
I think this line isn't needed

if it is, we have to do it a different way because it will corrupt any query running on the main thread at the same time (probably by passing the sql handle in or something - but the pet list is global anyways so it should be loaded already)